plugins/github-deployments/src/components/GithubDeploymentsTable/GithubDeploymentsTable.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Table, TableColumn } from '@backstage/core';
+import { GithubDeployment } from '../../api';
+import moment from 'moment';
+import { Box, Typography, Link, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+const lastUpdated = (start: string): string => moment(start).fromNow();
+
+const State = ({ value }: { value: string }) => {
+  const colorMap: Record<string, string> = {
+    PENDING: 'orange',
+    IN_PROGRESS: 'orange',
+    ACTIVE: 'green',
```

Comment:
The `BackstageTheme` actually exposes `theme.palette.status.*` with a number of values.

But, even more interestingly, the concept of "status dots" is also available from core in the form of the `<StatusWarning />`, `<StatusPending />` etc components that you should use to get a consistent look-and-feel

Reply:
👍 Nice - Updated.